front-end-tools/README.md

```diff
@@ -0,0 +1,81 @@
+# Developer Front End Tools
+
+The front end contains useful tools for developers:
+
+- Upload and deploy a smart contract module to chain.
+- Initialize a smart contract module on chain.
+
+Only the browser wallet (no walletConnect) is supported in the first version.
+
+## Prerequisites
+
+-   Browser wallet extension must be installed in Chrome browser.
+
+## Running the front end
```

Comment:
This section and below is meant for people developing this, right?
Should we have a section for how to use the docker image that we have pushed to docker hub?